db/data/questions_data.js

```diff
@@ -50,6 +23,13 @@ const questions_data = [
     answerType: "musicAnswers",
     allowedResponses: "0",
   },
+  {
+    question: "How old is too old?",
+    choices: ["2010's", "2000's", "1990's", "1980's", "no such thing as too old"],
```

Comment:
Did we agree on keeping the last value  ("no such thing as too old") in the array of choices?

Reply:
I believe so, though it may be more of a visual thing than anything. I believe our logic regarding this will just filter nothing if no year is present. This seems good for UX and will be necessary if we require answers to every question.